### PR TITLE
feat: add section kibitalk 070223

### PR DESCRIPTION
**Pull request by MahePrime**

### CODE REVIEW

File: 07022023/web-project/test/pageobjects/signup.page.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	get inputGender() { return $$("[name='sex']"); }
+	get linkLearnMore() { return $("#non-users-notice-link"); }
+	get buttonSignUp() { return $("(//button)[1]"); }
+
+	async open() {
+		await super.open("");
+	}
+
+	async chooseDob(value: string) {
+		const selector = $("//option[text()='"+value+"']");
```
Comment (mananrangkuti) [inline 07022023/web-project/test/pageobjects/signup.page.ts]: fix indentation

File: 07022023/web-project/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
```
Comment (mananrangkuti) [inline 07022023/web-project/tsconfig.json]: update all indentation to 2

**Review by pushm0v — APPROVED** [review]

File: 07022023/web-project/test/pageobjects/signup.page.ts (hunk 1 above)
Comment (MahePrime) [reply 07022023/web-project/test/pageobjects/signup.page.ts]: Mba @qonitatanjung tolong yak qaqaaa, pake branch ini aja Mba ambil dr remote

Comment (qonitatanjung) [reply 07022023/web-project/test/pageobjects/signup.page.ts]: done

File: 07022023/web-project/tsconfig.json (hunk 2 above)
Comment (qonitatanjung) [reply 07022023/web-project/tsconfig.json]: done

**Review by mananrangkuti — APPROVED** [review]

**Pull request merged**